lua/catppuccin/init.lua

```diff
@@ -80,20 +80,44 @@ function M.compile()
 	M.flavour = _flavour -- Restore user flavour after compile
 end
 
+---@param flavour? string
+local function get_flavour(flavour)
+	if flavour and not M.flavours[flavour] then
+		vim.notify(
+			string.format(
+				"Catppuccin (error): Invalid flavour '%s', flavour must be 'latte', 'frappe', 'macchiato' or 'mocha'",
+				flavour
+			),
+			vim.log.levels.ERROR
+		)
+		flavour = "mocha"
+	end
+
+	if flavour then
+		return flavour
+	elseif vim.g.colors_name and vim.g.colors_name == "catppuccin" then
```

Comment:
Shouldn't 

```lua
elseif vim.g.colors_name == "catppuccin" then
```

be sufficient

Reply:
you're right, didn't know lua is able to compare nil directly